TestFoundation/TestStream.swift

```diff
@@ -55,9 +55,8 @@ class TestStream : XCTestCase {
         let message: NSString = "Hello, playground"
         let messageData: Data  = message.data(using: String.Encoding.utf8.rawValue)!
         //Initialiser with url
-        let testFile = createTestFile("testFile_in.txt", _contents: messageData)
-        if testFile != nil {
-            let url = URL(fileURLWithPath: testFile!)
+        if let testFile = createTestFile("testFile_in.txt", _contents: messageData) {
```

Comment:
While these are all improvements, I think perhaps we could do better:

```swift
guard let testFile = createTestFile("testFile_in.txt", _contents: messageData)  else {
   XCTFail("Unable to create temp file")
   return
}
...
```

`guard` is great for reducing the indentation and putting failure code near the test.